app/components/Taps/TapConfiguration/Adwords/index.js

```diff
@@ -41,42 +41,30 @@ import {
   Row
 } from 'reactstrap';
 import { ipcRenderer } from 'electron';
-import type { tapPropertiesType } from '../../../../utils/shared-types';
+import type {
+  TapAdwords,
+  UpdateTapField,
+  AdwordsState,
```

Comment:
Agreed.

For this case the state type is also being used in the validate function located in util, instead of copy pasting the same type I decided to have it in shared types where both the validate function and the component can reference it.